lib/element.rb

```diff
@@ -266,7 +280,9 @@ def find_element(by, locator)
   # @return [Array] elements
   #
   def find_elements(by, locator)
-    element.find_elements(by, locator)
+    elements = element.find_elements(by, locator)
+    gridium_elements = elements.map {|element| Element.new("Child of #{@name}", by, locator, @element)}
```

Comment:
You can get rid of these 2 lines and just have `elements.map {|element| Element.new("Child of #{@name}", by, locator, @element)}` as the last returned value in the method